chef/data_bags/crowbar/migrate/nova_dashboard/026_use_keystone_session_timeout.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  keystone_instance = a['keystone_instance']
+  proposal_obj = ProposalObject.find_proposal("keystone", keystone_instance)
+  keystone_timeout = proposal_obj["attributes"]["keystone"]["token_expiration"]/60
```

Comment:
What happens if the keystone migration that adds this attribute has not run before? Based on what we've seen for neutron, it seems we should deal with that case.